lib/mixins/property-effects.html

```diff
@@ -506,36 +506,29 @@
    * @param {*} value Current value of property
    * @param {*} old Previous value of property
    * @param {Object} info Effect metadata
+   * @param {boolean} hasPaths True with `props` contains one or more paths
+   * @param {Array} nodes List of nodes associated with `nodeInfoList` template
+   *   metadata
    * @private
    */
-  function runBindingEffect(inst, path, props, oldProps, info, hasPaths) {
-    let value;
-    let node = inst.__templateNodes[info.index];
+  function runBindingEffect(inst, path, props, oldProps, info, hasPaths, nodeList) {
+    let node = nodeList[info.index];
+    let binding = info.binding;
+    let part = info.part;
     // Subpath notification: transform path and set to client
     // e.g.: foo="{{obj.sub}}", path: 'obj.sub.prop', set 'foo.prop'=obj.sub.prop
-    if (hasPaths && (path.length > info.value.length) &&
-        (info.kind == 'property') && !info.isCompound &&
-        node.__propertyEffects && node.__propertyEffects[info.name]) {
+    if (hasPaths && part.source && (path.length > part.source.length) &&
```

Comment:
Add a comment explaining why path processing is not included with `_evaluateBinding`.